inst/js/init.js

```diff
@@ -1,5 +0,0 @@
-// This file contains functions that should be executed at the start of each session,
-// not included in the original HTML
-
-// this code alows the show R code "copy to clipbaord" button to work
-var clipboard = new ClipboardJS(".btn[data-clipboard-target]");
```

Comment:
Just saying that `show_rcode_modal` uses this. It is still soft (not hard) deprecated

Reply:
👍🏽 Let me move this dependency to just the `show_rcode_modal()` for now.